app/build.gradle

```diff
@@ -79,16 +89,10 @@ android {
         full {
             dimension 'license'
         }
-        dw {
-            dimension 'testing'
-            applicationId 'com.dw.ooniprobe'
-            resValue "string", "APP_ID", 'com.dw.ooniprobe'
-            resValue "string", "APP_NAME", "News Media Scan"
-            buildConfigField 'String', 'BASE_SOFTWARE_NAME', '"news-media-scan-android"'
-            buildConfigField "boolean", "SHOW_DISABLED_CARDS", "false"
-            //versionName '0.0.1'
-            //versionCode 1
-        }
+    }
+    applicationVariants.all { variant ->
+        variant.resValue "string", "APP_ID", "\"${applicationId}\""
+        // TODO(aanorbel): Use similar solution to set `APP_NAME` in the format `provider` `testing`.
```

Comment:
Do you want to do this as part of the same PR or as part of another PR? (I am flexible)

Reply:
This PR